cmd/apps/openfaas_app.go

```diff
@@ -148,6 +157,36 @@ func MakeInstallOpenFaaS() *cobra.Command {
 		if err != nil {
 			return err
 		}
+		autoscaler, err := command.Flags().GetBool("autoscaler")
+		if err != nil {
+			return err
+		}
+		dashboard, err := command.Flags().GetBool("dashboard")
+		if err != nil {
+			return err
+		}
+
+		if dashboard {
+			// Private key
+			priv, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+			ecder, _ := x509.MarshalECPrivateKey(priv)
+			privOut := bytes.Buffer{}
+			pem.Encode(&privOut, &pem.Block{Type: "EC PRIVATE KEY", Bytes: ecder})
+
+			// Public key
+			pub := &priv.PublicKey
+			pubder, _ := x509.MarshalPKIXPublicKey(pub)
+			pubOut := bytes.Buffer{}
+			pem.Encode(&pubOut, &pem.Block{Type: "PUBLIC KEY", Bytes: pubder})
+
+			secretData := []types.SecretsData{
+				{Type: types.StringLiteralSecret, Key: "key", Value: privOut.String()},
+				{Type: types.StringLiteralSecret, Key: "key.pub", Value: pubOut.String()},
+			}
+
+			dashboardJWT := types.NewGenericSecret("dashboard-jwt", namespace, secretData)
```

Comment:
What happens when you install this arkade app multiple times?

I would expect any existing secret to be kept / honoured

Reply:
I think we would do nothing if the secrets exist.

I can't remember how the openfaas app handles this

Reply:
The secret is not recreated if it already exists. This is how it is handled for the basic-auth secret as well. Instead a message is printed.

<img width="1018" alt="Screenshot 2022-09-05 at 09 16 04" src="https://user-images.githubusercontent.com/16267532/188398512-7ed53930-d561-40eb-bd27-0a6a16334598.png">